File: modules/01-basic-syntax/go/src/modules/references.go
package modules

import (
	"fmt"
	"myapp/src/utils"
)

// References
func References() {
	utils.PrintH3("References")

	var x int32 = 10;
	var ref_x *int32 = &x;
	*ref_x = 20;
	fmt.Printf("x: %d\n", x);
}

File: modules/01-basic-syntax/rust/src/modules/references.rs
use crate::utils;

// References
pub fn references() {
    utils::print_h3("References");

    let mut x: i32 = 10;
    let ref_x: &mut i32 = &mut x;
    *ref_x = 20;
    println!("x: {x}");
}
